### ISSUE TITLE
Save Parameter key and values

### ISSUE THREAD

**Issue by dwmorrison33** [groveco/django-sql-explorer#482]
I am curious if there is a way to save a query parameters value for later use instead of having to re-enter it each time one goes to the `Query Detail` page? I only ask because I have not been able to find an answer to that with Google and I do not see anything in the codebase to do it, but maybe I am missing something. Thank you

**Comment by marksweb** [comment]
@dwmorrison33 could you provide an example of what you mean please?

**Comment by dwmorrison33** [comment]
@marksweb Thank you for your response, below is an example. I create a new query with `id` as the parameter and execute it:
<img width="1590" alt="Screen Shot 2022-08-27 at 3 46 31 PM" src="https://user-images.githubusercontent.com/13126248/187045833-0f0718a3-3e21-4265-b82a-c5d347cb2da4.png">

After navigating back to the Query List page and clicking again on this query, the parameter `id` no longer has the value `10000` populated:
<img width="1586" alt="Screen Shot 2022-08-27 at 3 46 47 PM" src="https://user-images.githubusercontent.com/13126248/187045879-dab171e5-e8ee-4f16-8e7e-e4e25043fecf.png">

Is there a way to persist this value? Also, on a side note, I am using version `1.1.3` of your package since my company is on Django 1.11 and that is the last version that is compatible.

**Comment by marksweb** [comment]
@dwmorrison33 Right, I'm with you - and I thought that was what you meant. I'm fairly sure the app can't do that, at least as it is now.

My suggestion would be to create bookmarks with the query params defined in them.

**Comment by dwmorrison33** [comment]
> @dwmorrison33 Right, I'm with you - and I thought that was what you meant. I'm fairly sure the app can't do that, at least as it is now.
> 
> My suggestion would be to create bookmarks with the query params defined in them.

Would you mind if I implemented a feature to save the query Params so that we can save a default value and then put in a pull request, as saving bookmarks will become a bit laborious across the organizations users who are using these queries each day. I just want to check before I work on this feature and you all not wanting it, but I think it would be useful to others as well.

**Comment by marksweb** [comment]
@dwmorrison33 Yeah that sounds like it'd be really useful.

Create a branch from the 1.1.3 tag and when you come to a PR, target this new branch I've setup; https://github.com/groveco/django-sql-explorer/tree/release/1.1.4

Then I can do a 1.1.4 release and then, hopefully, merge it to the main branch for it to benefit the users of the latest version as well 👍

**Comment by dwmorrison33** [comment]
@marksweb I had a chance to work on the changes yesterday and finished them up this morning, but do not have permission to push up the code to the repo. Is there another way to push the code so that I can create a PR and request to merge into https://github.com/groveco/django-sql-explorer/tree/release/1.1.4?

**Comment by marksweb** [comment]
@dwmorrison33 ok, so not having permission is correct.

The workflow for contributing is among the lines of...

- fork the repo
- clone your fork to your local machine
- add the main repo as a remote (this allows you to pull changes in to your fork and base branches from upstream branches like we want here)
- create a branch for your contribution and remember to check for guidelines on the repo
- once complete and all changes are pushed to your fork github will allow you to create a pull request from the branch.
- the PR will be reviewed & merged if accepted

**Comment by chrisclark** [comment]
See my comment on the PR. I believe this is already possible via this feature (as per the docs):

> You can combine both a default and label to your parameter but you must start with the label: $$paramName|label:defaultValue$$.
